README.md

```diff
@@ -256,12 +256,6 @@ packages:
 
     - package: dbt-labs/spark_utils
       version: [">=0.3.0", "<0.4.0"]
-
```

Comment:
Can you bump the suggested release range in for hubspot in the README as well

Reply:
yup!